### ISSUE TITLE
Offset

### ISSUE THREAD

**Issue by TattieBoJangle** [mklabs/ue4-targetsystemplugin#36]
Hi can anyone add a left, right offset please i have tried using paid lock on plugins and they are not that good, this one works perfect but i need to have a left right offset like the way it has up/down for my project to work properly thanks.

**Comment by mklabs** [comment]
The plugin exposes some events you can listen to whenever you are targetting an actor, and untargetting it. It should be able to implement your desired offset in Blueprint by leveraging those events.

https://github.com/mklabs/ue4-targetsystemplugin/wiki/Blueprint-Functions-and-Events#events

OnTargetLockOn and OnTargetLockOff

**Comment by TattieBoJangle** [comment]
Yeah shame i don't have a clue what to do or i would try it there is one on the marketplace that allows changing the left right offset but it is garbage compared to this.

**Comment by vekien** [comment]
> Yeah shame i don't have a clue what to do or i would try it there is one on the marketplace that allows changing the left right offset but it is garbage compared to this.

If you had not figured this out, you can add a Scene Component and attach your Spring Arm > Camera to this, then offset that scene component and it'll all work fine. Here is mine:

![image](https://github.com/mklabs/ue4-targetsystemplugin/assets/270800/996036c9-f2c3-4f2b-9553-04c1b1a8e69d)

I often increase the LockOnLocation for taller mobs, but I can also do Left/Right.
